Assets/HoloToolkit/Input/Scripts/Gaze/GazeManager.cs

```diff
@@ -189,7 +203,7 @@ private void UpdateGazeInfo()
                     newGazeNormal = Stabilizer.StableRay.direction;
                 }
 
-                Ray = new Ray(newGazeOrigin, newGazeNormal);
+                FirstRay = new RayStep(newGazeOrigin, newGazeOrigin + newGazeNormal);
```

Comment:
Seeing as this is a struct you don't need to use the new keyword each time.  Setting the new values should be enough.